silhouette/app/com/mohiva/play/silhouette/api/util/DefaultEndpointHandler.scala

```diff
@@ -12,29 +12,29 @@ import scala.concurrent._
 object DefaultEndpointHandler extends Controller {
 
   /**
-   * Handles forbidden requests in a default way.
+   * Handles unauthorized requests in a default way.
    *
-   * The HTTP status code of the response will be 403 Forbidden, complying with
-   * [[https://tools.ietf.org/html/rfc2616#section-10.4.4 RFC 2616]].
+   * The HTTP status code of the response will be 401 Unauthorized, complying with
+   * [[https://tools.ietf.org/html/rfc2616#section-10.4.2 RFC 2616]].
    *
    * @param request The request header.
-   * @return A response indicating that access is forbidden.
+   * @return A response indicating that user authentication is required.
    */
-  def handleForbidden(implicit request: RequestHeader): Future[Result] = {
-    produceResponse(Forbidden, "silhouette.not.authorized")
+  def handleUnauthorized(implicit request: RequestHeader): Future[Result] = {
```

Comment:
Play use 'on' instance of 'handle' as 'onStart', 'onStop', 'onError', ...
So i suggest to rename to onUnauthorized